### DISCUSSION TITLE
물 만드는 방법

### DISCUSSION THREAD

**Discussion by yj098** [naverz/zepeto-studio-kor#1110] [5. Q&A]
유니티에서 물을 만들어 보고 싶은데 유튜브 강좌는 거의 URP 사용하네요 ㅠ
유니티에서 어떻게 물을 만드나요ㅠ?

**Comment by YSHFM** [comment]
https://bornsoul.tistory.com/16

해당 글 참고하여 저는 물 그래픽을 만들었습니다.

https://www.mettit.com/log/view/LOG20220729131912463 이러한 물 그래픽을 만들 수 있습니다.

**Reply by yj098** [reply]
와~~~!! 감사합니다 :) :)!!!

**Reply by yoon-hw22** [reply]
혹시 c#으로 만든 커스텀 쉐이더도 제페토에서 적용이 되나요?

**Reply by YSHFM** [reply]
> 혹시 c#으로 만든 커스텀 쉐이더도 제페토에서 적용이 되나요?

유니티 쉐이더 스크립트는 c#하고 다른 언어를 사용합니다. 

제페토에서 빌드 시 c#스크립트는 포함되지 않습니다.
